clients/src/main/java/org/apache/kafka/clients/admin/OffsetSpec.java

```diff
@@ -70,4 +72,21 @@ public static OffsetSpec maxTimestamp() {
         return new MaxTimestampSpec();
     }
 
+    /**
+     * Used to retrieve the offset with the local log start offset,
+     * log start offset is the offset of a log above which reads are guaranteed to be served
+     * from the disk of the leader broker, when Tiered Storage is not enabled, it behaves the same
+     * as the earliest timestamp
```

Comment:
nit: Used to retrieve ~~the offset with~~ the local log start offset[.] 
[Local] log start offset is the offset of a log above which reads are guaranteed to be served from the disk of the leader broker.

Note: When tiered storage is not enabled, it behaves the same as retrieving the earliest timestamp offset.